src/overlays/fbdemos/ovl_fbdemo_wipe5/z_fbdemo_wipe5.c

```diff
@@ -16,37 +18,133 @@ void TransitionWipe5_SetColor(void* thisx, u32 color);
 void TransitionWipe5_SetEnvColor(void* thisx, u32 color);
 s32 TransitionWipe5_IsDone(void* thisx);
 
-#if 0
 const TransitionInit TransitionWipe5_InitVars = {
-    TransitionWipe5_Init,
-    TransitionWipe5_Destroy,
-    TransitionWipe5_Update,
-    TransitionWipe5_Draw,
-    TransitionWipe5_Start,
-    TransitionWipe5_SetType,
-    TransitionWipe5_SetColor,
-    TransitionWipe5_SetEnvColor,
-    TransitionWipe5_IsDone,
+    TransitionWipe5_Init,     TransitionWipe5_Destroy,     TransitionWipe5_Update,
+    TransitionWipe5_Draw,     TransitionWipe5_Start,       TransitionWipe5_SetType,
+    TransitionWipe5_SetColor, TransitionWipe5_SetEnvColor, TransitionWipe5_IsDone,
 };
 
-#endif
+void TransitionWipe5_Start(void* thisx) {
+    TransitionWipe5* this = THIS;
+
+    this->isDone = false;
+    switch (this->unk_12) {
+        case 0:
+            this->unk_8 = 0.02f;
+            break;
+        case 1:
+            this->unk_8 = 0.05f;
+            break;
+        default:
+            this->unk_8 = 0.2f;
+            break;
+    }
+    switch (this->unk_11) {
+        case 0:
+            this->primColor.rgba = 0x000000FF;
+            break;
+        case 1:
+            this->primColor.rgba = 0xA0A0A0FF;
+            break;
+        default:
+            this->primColor.rgba = 0xA0A0A0FF;
+            break;
+    }
+}
+
+void* TransitionWipe5_Init(void* thisx) {
+    TransitionWipe5* this = THIS;
+
+    bzero(this, sizeof(TransitionWipe5));
```

Comment:
```suggestion
    bzero(this, sizeof(*this));
```
is probably better.

Reply:
We have been using the type explicitly instead of this on `sizeof` everywhere else